entrypoint.sh

```diff
@@ -39,6 +39,14 @@ if [[ "${INPUT_REPORTER}" = 'github-pr-review' ]]; then
     -level="${INPUT_LEVEL}" \
     -fail-on-error="${INPUT_FAIL_ON_ERROR}" \
     ${INPUT_REVIEWDOG_FLAGS} || reviewdog_exit_val="$?"
+
+  # regenerate black_check_output that command same as non-github-pr-review.
+  # because after process need normal generated black_check_output value.
+  # read and drop exit code
+  black_exit_val2="0"
+  # shellcheck disable=SC2086,SC2034
+  black_check_output="$(black --check . ${INPUT_BLACK_ARGS} 2>&1)" ||
```

Comment:
@tsuyoshicho I think this can be changed to `black_check_output="$(black --check . ${INPUT_BLACK_ARGS} 2>&1)" || true` since we don't use the exit code.